atomic

```diff
@@ -431,6 +431,20 @@ def create_parser(atomic):
     scan_group.add_argument("--images", default=False, action='store_true', help=_("scan all images (excluding intermediate layers)"))
     scan_group.add_argument("--containers", default=False, action='store_true', help=_("scan all containers"))
 
+    # atomic ps
+    pss = subparser.add_parser(
+        "ps", help=_("List containers"),
+        epilog="show the installed containers")
+    pss.set_defaults(func='ps')
```

Comment:
man docker shows
       List the containers in the local repository. By default  this  shows  only
       the running containers.

Should we add to the epilog something about showing only running containers.